components/org.wso2.carbon.identity.conditional.auth.functions.session/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/session/function/IsValid.java

```diff
@@ -27,7 +27,7 @@
  * Function definition for verify some defined requirement is fulfilled.
  */
 @FunctionalInterface
-public interface IsValidFunction {
+public interface IsValid {
```

Comment:
Shall we have it as 'IsWithinSessionLimitFunction', because this is specific to this component and function

Reply:
fixed in the new commits